server/services/mcp/api/mcp.controller.js

```diff
@@ -0,0 +1,20 @@
+const asyncMiddleware = require('../../../api/middlewares/asyncMiddleware');
+const { proxy: proxyFunction } = require('../lib/mcp.proxy');
+
+module.exports = function MCPController(mcpHandler) {
+  /**
+   * @api {post} /api/v1/service/mcp/proxy Connect to MCP bridge
+   * @apiName proxy
+   * @apiGroup MCP
+   */
+  async function proxy(req, res) {
+    return proxyFunction(req, res, mcpHandler);
+  }
+
+  return {
+    'post /api/v1/service/mcp/proxy': {
+      authenticated: true,
```

Comment:
You are not verifying that the "mcp" scope is present

Maybe we could add a "scope" property here and in the `server/api/setupRoutes.js` file, verify that the scope is present (by creating an instance of AuthMiddleware with the correct scopes)

Reply:
The scope still needs to be an array! 

API Keys aren't used yet, but we already use scopes for user sessions, and we already have clients with multiple scopes :)